gstd/Cargo.toml

```diff
@@ -7,6 +7,9 @@ license = "GPL-3.0"
 
 [dependencies]
 dlmalloc = { git = "https://github.com/nikvolf/dlmalloc-rs", features = ["global"] }
+scale-info = { version = "0.10.0", default-features = false, features = ["derive", "serde"] }
```

Comment:
Nothing of this should be a dependency of gstd 

this should go to gstd-meta or similar